misc/ninja_syntax.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/python
```

Comment:
I recommend against removing the shebang.

Reply:
I removed it in this case because this file isn't executable and this file on its own doesn't actually run anything if you it was executable. I can add it back if you're thinking the documentation aspect is worth it?

Reply:
Personally I think consistency is best.

Reply:
Done!